definitions/apm-application/relationships.yml

```diff
@@ -0,0 +1,30 @@
+relationships:
+  # Select entityGuid, peer.hostname from Span where peer.hostname is not null and category =
```

Comment:
Rule for location of definition is: Define it in the file where the data point is originating

Reply:
Add documentation